pybossa/view/projects.py

```diff
@@ -3129,6 +3129,10 @@ def transfer_ownership(short_name):
         return handle_content_type(response)
 
 
+def is_user_enabled_assigned_project(self, user, project):
```

Comment:
```self``` is not needed. We can also define this function inside ```def coowners(short_name)``` if it is only used by this function. The good thing to define it inside the function is that it can access "project" (without passing it as a parameter).

Reply:
Removed `self` https://github.com/bloomberg/pybossa/pull/761/commits/524f04f170a0d4fc49a8fd1f76de3502660503f6

The refactored method is used by two methods: coowners() and contact().